src/components/Calculator.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import Button from './ButtonSkeleton';
+
+const operators = ['÷', 'x', '-', '+', '='];
+const digits = ['AC', '+/-', '%', '7', '8', '9', '4', '5', '6', '1', '2', '3', '0', '.'];
+
+class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="calc-container">
+        <div className="calc-input">0</div>
```

Comment:
- [ ] Kindly ensure your design matches the design given, in this section your background is different from the background in the design given:
`design given`
![image](https://user-images.githubusercontent.com/61408860/182685903-9c424d6e-50fc-4834-ac6b-7a212be0ebc9.png)

`your design`
![image](https://user-images.githubusercontent.com/61408860/182686007-e2a1b99a-1874-401d-80d3-8792a0116d72.png)

Reply:
Done! Thanks for the review!